test/unit/converter.test.js

```diff
@@ -120,7 +123,7 @@ describe('Converter tests', function () {
     it('should generate a collection for a valid SDL schema with a union type self referencing', function (done) {
       convert({ type: 'string',
         data: selfRefSchema
-      }, {}, function (error, result) {
+      }, { depth: 3 }, function (error, result) {
```

Comment:
@VShingala What's the default depth limit if we don't specify it here?

Reply:
@webholik It's 1. https://github.com/postmanlabs/graphql-to-postman/blob/589d5c8ba4eee71289a5ed316596ad9d2e3733cf/lib/index.js#L192